### ISSUE TITLE
recommendation: please provide dummy / fake authentication too

### ISSUE THREAD

**Issue by faizalami** [typicode/jsonplaceholder#107]
I think it will very useful if jsonplaceholder also provide a dummy / fake authentication for testing and learning

**Comment by jong19** [comment]
> I think it will very useful if jsonplaceholder also provide a dummy / fake authentication for testing and learning

Yes, that would be very useful to someone like me as a beginner.
